### ISSUE TITLE
Windows seems to be cropped beyond a given X and Y screen coordinate

### ISSUE THREAD

**Issue by poliveira81** [apemberton/outsystems-studio-mac#22]
If I move the window the absolute crop point (from a screen perspective) remains the same.
See screenshot attached
![screen shot 2018-04-12 at 18 16 51](https://user-images.githubusercontent.com/20322186/38693411-b3d289a8-3e7e-11e8-9606-a4ec17621c43.png)

**Comment by apemberton** [comment]
@poliveira81 can you also retest in latest 0.16 release? I can't seem to reproduce this one.

**Comment by poliveira81** [comment]
I think I better understood what's wrong now.
I had started on a single screen and then connected and additional monitor.
The window is cropped to the original resolution of the first screen.

If when I start the Studio I have it in mirroring with the external monitor it will work just fine.

**Comment by apemberton** [comment]
Yes. This is known and related to the other issue around dual monitors.
